tests/functional/firefox/os/test_tv.py

```diff
@@ -0,0 +1,34 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.firefox.os.tv import TVPage
+
+
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_next_previous_buttons(base_url, selenium):
+    page = TVPage(base_url, selenium).open()
+    assert not page.is_previous_enabled
+    page.click_next()
+    screen = page.wait_for_screen_to_display('tv2')
+    assert page.is_previous_enabled
+    assert screen.is_displayed
+    page.click_previous()
+    screen = page.wait_for_screen_to_display('tv1')
+    assert not page.is_previous_enabled
+    assert screen.is_displayed
```

Comment:
Instead of maintaining a dictionary of the screens, I would propose having `screens` and `thumbnails` as properties. These would be regions for interacting with and determining if they're displayed. The test would then look something like:

```python
def test_next_previous_buttons(base_url, selenium):
    page = TVPage(base_url, selenium).open()
    assert page.screen[0].is_displayed
    assert not page.is_previous_enabled
    page.click_next()
    assert page.screen[1].is_displayed
    page.click_previous()
    assert not page.is_previous_enabled
    assert page.screen[0].is_displayed
```

We could then even add coverage by testing all screens:

```python
def test_next_previous_buttons(base_url, selenium):
    page = TVPage(base_url, selenium).open()
    assert not page.is_previous_enabled
    for screen in page.screens[:-1]:
        assert screen.is_displayed
        page.click_next()
    assert not page.is_next_enabled
    for screen in reversed(page.screens)[:-1]:
        screen.is_displayed
        page.click_previous()
    assert not page.is_previous_enabled
    assert page.screens[0].is_displayed
```

This could similarly apply to the thumbnails. If you used `for i in range(len(page.screens[:-1]))` then you could reference both screens and thumbnails by a common index, allowing you to check the correct thumbnail is highlighted when using the next/previous buttons, and vice versa.